x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -244,6 +247,13 @@ export const AlertsTableComponent: FC<DetectionEngineAlertTableProps> = ({
     [dispatch, tableId, alertTableRefreshHandlerRef, setQuery]
   );
 
+  // Highlight rows with building block alerts
+  const rowClassMapper: AlertsTableStateProps['rowClassMapper'] = useCallback(
+    (alert: Alert) =>
+      alert[ALERT_BUILDING_BLOCK_TYPE] ? 'alertsTable-rowBackgroundHighlighted' : '',
```

Comment:
is there a way to import this class name from `alerts_table/index.scss`?

Reply:
like its done here: `x-pack/plugins/canvas/shareable_runtime/components/page.tsx`

Reply:
Ohhh, that's neat. I didn't know we could use CSS modules in Kibana.

Though, I just tried to transform it to a CSS module and it doesn't seem to work. Will ask the core team if I need to change something to get CSS modules to work properly in `security_solution` 👍

Reply:
Moved the class to the alert table package now (`plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.scss`).

Sadly I couldn't make CSS modules work in that context (or in any other for that matter)